dts/bindings/qspi/st,stm32-qspi.yaml

```diff
@@ -30,19 +30,17 @@ properties:
       required: true
 
     dmas:
-      required: true
       description: |
-        DMA channel specifier. Specifier will have a phandle
-        reference to the dma controller, the channel number, the slot number,
-        channel configuration and finally features.
+        Optional DMA channel specifier. If DMA should be used, specifier should
+        hold a phandle reference to the dma controller, the channel number,
+        the slot number, channel configuration and finally features.
 
         For example dmas for TX/RX on QSPI
            dmas = <&dma1 5 5 0x0000 0x03>;
 
     dma-names:
-      required: true
       description: |
-        DMA channel name. Expected value "tx_rx".
+        DMA channel name. If DMA should be used, expected value is "tx_rx".
```

Comment:
Can this be refactored so it's not changing what was introduced in an earlier commit, or is that too much hassle?  ("It's too much hassle" is an acceptable answer.)

Reply:
What could be done is squash the latest commits that introduced **optional** dma.
I kept those separate in order not confusing on going reviews, but I can squash if required.